docs/deprecated_wiki/Setting-Up-Git.md

```diff
@@ -1,86 +0,0 @@
-# Terminology
```

Comment:
Git configuration is not in the scope of LORIS documentation. This can be found elsewhere online. If we want to provide this information for new devs it should be part of internal documentation such as "LORIS 101", not in the repository for general use.